app/authenticate/forms.py

```diff
@@ -0,0 +1,15 @@
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth.models import User
```

Comment:
This is a minor point, but it was raised by py-linter when I ran it against the code. A better way to reference the user model is below:
```suggestion
from django.contrib.auth import get_user_model
```

then use that function below:

```
    class Meta:
        model = get_user_model()
```

The reason for this is that `get_user_model()` gets the actual user model as set in `app.settings` or the default `user` model if a custom model is not being used. So if you wanted to use a custom `user` model in the future, you wouldn't need to alter any of the imports in other files.